pvacseq/server/controllers/staging.py

```diff
@@ -430,3 +431,18 @@ def check_allele(allele):
         if line.strip() == allele:
             return True
     return False
+
+# takes in comma delimited string of prediction algorithms,
+# returns map of algorithms to valid alleles for that algorithm
+def valid_alleles(prediction_algorithms):
+    valid_allele_list = {}
+    for algorithm in prediction_algorithms.split(","):
+        alleles = subprocess.run(['pvacseq', 'valid_alleles', '-p', algorithm], stdout=subprocess.PIPE).stdout.decode('utf-8')
```

Comment:
Instead of making a system call to  `pvacseq valid_alleles` you can just call `PredictionClass.all_valid_allele_names()` directly. It will return an array of all valid alleles names.

If you are concerned about only getting alleles for the set of prediction algorithms that the user chose you can do the following:
```
prediction_class = globals()[algorithm]
alleles = prediction_class().valid_allele_names()
```
This returns an array already.